### PR TITLE
Fixes for EL8 FIPS and switching deprecated functionality

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: katello_certs_tools/sslToolConfig.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
 SERVER_OPENSSL_CNF_NAME = 'katello-server-openssl.cnf'
 
 MD = 'sha256'
-CRYPTO = '-des3'
+CRYPTO = '-aes256'
```
Comment (evgeni) [inline katello_certs_tools/sslToolConfig.py:50]: how does this affect existing installs? are they left untouched, are they upgraded, are they broken? ;)

File: katello-certs-gen-rpm [hunk 2]
```diff
@@ -340,8 +340,6 @@ RPMOPTS="--define \"_topdir $RPM_BUILD_DIR\"\
  --define '_specdir     %{_topdir}'\
  --define '_rpmdir      %{_topdir}'\
  --define '_srcrpmdir   %{_topdir}'\
- --define '_source_filedigest_algorithm md5'\
- --define '_binary_filedigest_algorithm md5'\
```
Comment (evgeni) [inline katello-certs-gen-rpm:344]: this should have been removed a long time ago (when we dropped EL5 client support)

File: katello_certs_tools/sslToolConfig.py (hunk 1 above)
Comment (ehelms) [reply katello_certs_tools/sslToolConfig.py:50]: This is only used when generating the CA private key, so I think existing environments should be untouched. I can test this on a upgrade to be sure. I am not clear what happens with an existing des3 based key on a FIPS enabled EL8 environment but thats a whole 'nother can o' worms.

Comment (ehelms) [reply katello_certs_tools/sslToolConfig.py:50]: For example, there is this guard in place https://github.com/Katello/katello-certs-tools/blob/master/katello_certs_tools/katello_ssl_tool.py#L156 and

Comment (evgeni) [reply katello_certs_tools/sslToolConfig.py:50]: I think unbreaking already broken setups can be left for the future

**Review by ekohl — APPROVED** [review]
This looks good to me. It would be good to look into the option to convert legacy private keys on upgrades in a follow up PR.

**Review by evgeni — APPROVED** [review]

**Pull request merged**